changelogs/fragments/72592-catch-broken-cowsay.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - catch the case that cowsay is broken which would lead to missing output
```

Comment:
```suggestion
  - display - catch the case that ``cowsay`` is broken which would lead to missing output (https://github.com/ansible/ansible/issues/72582).
```